frontend/src/pages/GenExamPage.js

```diff
@@ -94,65 +241,221 @@ const GenExamPage = () => {
+							<FormControl className={classes.boxes}>
+								<FormGroup >
+									<FormControlLabel labelPlacement="start" control={<Checkbox checked={options.randomSubSubject} icon = {<GreyRoundCheckbox/>} checkedIcon = {<RedRoundCheckmark/>} onChange={handleChangeRandomSubSubject} name="randomSubSubject"/>} label={<Typography variant = "h6">Aleatório</Typography>}/>
+									<FormControlLabel labelPlacement="start" control={<Checkbox checked={dictSubSubjects.geometry && !options.randomSubSubject} icon = {<GreyRoundCheckbox/>} checkedIcon = {<RedRoundCheckmark/>}  onChange={handleChangeSubSubjects} name="geometry"/>} label={<Typography variant = "h6">Geometria</Typography>}/>
+									<FormControlLabel labelPlacement="start" control={<Checkbox checked={dictSubSubjects.imaginary && !options.randomSubSubject} icon = {<GreyRoundCheckbox/>} checkedIcon = {<RedRoundCheckmark/>} onChange={handleChangeSubSubjects} name="imaginary"/>} label={<Typography variant = "h6">Imaginários</Typography>}/>
+									<FormControlLabel labelPlacement="start" control={<Checkbox checked={dictSubSubjects.statistics && !options.randomSubSubject} icon = {<GreyRoundCheckbox/>} checkedIcon = {<RedRoundCheckmark/>} onChange={handleChangeSubSubjects} name="statistics"/>} label={<Typography variant = "h6">Estatística</Typography>}/>
+									<FormControlLabel labelPlacement="start" control={<Checkbox checked={dictSubSubjects.probability && !options.randomSubSubject} icon = {<GreyRoundCheckbox/>} checkedIcon = {<RedRoundCheckmark/>} onChange={handleChangeSubSubjects} name="probability"/>} label={<Typography variant = "h6">Probabilidades</Typography>}/>
+								</FormGroup>
+							</FormControl>
+						</Grid>
+					</Grid> 
+
+					<Grid justifyContent="center" container > {/*Começar Button*/}
+						<Grid className={classes.button} item>
+							<NormalButton  fontSize={28} text="Começar" onClick={handleClick}/>
+						</Grid>
+						
+					</Grid>
+				</Grid>
+
+				{is1100pxScreen 
+
+				? 
+					<Grid container xs={12}  direction="column" justifyContent="center" alignItems="center">
+							<Grid container xs = {12}>
+								<Divider orientation= "horizontal" className={classes.divider} flexItem /> {/*Vertical Divider*/}
+							</Grid>
+							<Grid item xs = {12}>
+								<Typography variant="h6">ou</Typography> {/*Maybe put this in Bold */}
+							</Grid>
+							<Grid item xs = {12}>
+								<Typography className={classes.upperSideText} variant="h5">Deixa isso connosco</Typography>
+							</Grid>
+						</Grid>
+				: 
+
+					<Grid justifyContent="center" container xs = {1}>
+						<Divider orientation= "vertical" className={classes.divider} flexItem /> {/*Vertical Divider*/}
+					</Grid>
+				}
+	
+				<Grid container xs = {is1100pxScreen ? 12 : 5}> {/*right lower panel*/}
+					<Grid item>
+						<Typography variant = "h6" dangerouslySetInnerHTML={{__html: redString}}></Typography> {/* Best way to change a specific attribute in a string */}
```

Comment:
Does `<Typography>... <span style={{ color: theme.pallete.secondary .main}}>red text</span></Typography>` not work?

Reply:
Couldn't get it working, but not sure though. Might have done something wrong